packages/kbn-http-tools/src/ssl/ssl_config.ts

```diff
@@ -146,7 +145,7 @@ export class SslConfig {
   /**
    * Options that affect the OpenSSL protocol behavior via numeric bitmask of the SSL_OP_* options from OpenSSL Options.
    */
-  public getSecureOptions() {
+  public get secureOptions() {
```

Comment:
Nit: Feels like a getter, especially one that throws and recalculates the values on each access is less than ideal. I'd prefer this stay a method unless there's a better reason to switch to a getter that I'm not seeing.

Reply:
Did that to simplify the extracted interface at some point I believe before moving the `SslConfig` to the http-tools package. I agree that the getter is not ideal, will revert